docs/releases/2.1.txt

```diff
@@ -173,6 +173,9 @@ Models
 * The new :class:`~django.db.models.functions.Replace` database function
   replaces strings in an expression.
 
+* Expressions can now be made negative using the unary negative operator
+  before them.
```

Comment:
reads well, I'd just suggest that 'before them' can be removed, as there's no way to use the unary negative operator afterwards :)

Reply:
@adamchainz I'll fix this in a new commit. Should I squash it into this one ?

Reply:
please rebase and squash the commits into one as per https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/submitting-patches/#all-tickets

Reply:
Fixed.